src/limb_repo/environments/pybullet_env.py

```diff
@@ -0,0 +1,59 @@
+"""PyBullet environment for Limb Repositioning."""
+
+from dataclasses import dataclass
+
+import numpy as np
+import omegaconf
+import pybullet
+import pybullet_utils.bullet_client as bc
+from omegaconf import OmegaConf
+
+from limb_repo.environments.base_env import BaseEnv
+
+
+@dataclass
+class PyBulletConfig:
+    """Configuration for a PyBullet environment."""
+
+    use_gui: bool
+    real_time_simulation: bool
+    gravity: np.ndarray
+    dt: float
+
+
+class PyBulletEnv(BaseEnv):
+    """Pybullet environment for Limb Repositioning."""
+
+    def __init__(self, config: omegaconf.DictConfig):
+        # super().__init__()
+        # Create pybullet sim
+        if config.use_gui:
+            self.p = bc.BulletClient(
+                connection_mode=pybullet.GUI, options="--width=1000 --height=1000"
+            )
+        else:
+            self.p = bc.BulletClient(connection_mode=pybullet.DIRECT)
+
+        self.p.setGravity(*config.gravity)
+        self.p.setRealTimeSimulation(1 if config.real_time_simulation else 0)
+        self.p.setTimeStep(config.dt)
+        self.p.setPhysicsEngineParameter(
+            constraintSolverType=self.p.CONSTRAINT_SOLVER_LCP_DANTZIG,
+            globalCFM=0.000001,
+        )
+
+        self.dt = config.dt
+
+    @staticmethod
+    def parse_config(path_to_yaml: str) -> omegaconf.DictConfig:
+        """Parse a configuration file."""
+        config = omegaconf.DictConfig(OmegaConf.load(path_to_yaml))
```

Comment:
is this pattern recommended somewhere? I see it in the playground as well. it seems like there should be a better way...

Reply:
I did this to cast the output of load, which is typed `(DictConfig | ListConfig)` because i write that my function returns DictConfig. Otherwise, only doing load works fine. I could also `return cast(omegaconf.DictConfig, config)`. Or is there a better way to do this?

Reply:
if you do
```
assert isinstance(config, omegaconf.DictConfig)
```
would that work? usually mypy is smart enough for that to work.

my main comment though is that it seems like this parse_config function should live in a utilities file and be reused in all the places where you're parsing configs.